boards/phytec/mimx93_phyboard_nash/doc/index.rst

```diff
@@ -0,0 +1,182 @@
+.. _mimx93_phyboard_nash:
+
+PhyBOARD Nash (NXP i.MX93)
+##########################
+
+Overview
+********
+
+The phyBOARD Nash is based on the phyCORE-i.MX93 SoM is based on the NXP i.MX93
+SoC. It features common industrial interfaces and can be used as a reference for
+development or in the final product. It is an entry-level development board,
+which helps developers to get familiar with the module before investing a large
+amount of resources in more specific designs.
+
+i.MX93 MPU is composed of one cluster of 2x Cortex-A55 cores and a single
+Cortex®-M33 core. Zephyr OS is ported to run on one of the Cortex®-A55 core as
+well as the Cortex®-M33 core.
+
+- Memory:
+
+   - RAM: 512 MB - 2GB LPDDR4
+   - EEPROM: 4 kB - 32 kB
+   - eMMC: 8 GB - 256 GB
+
+- Interfaces:
+
+   - Ethernet: 2x 10/100BASE-T (1x TSN Support)
+   - USB: 2x 2.0 Host / OTG
+   - Serial: 1x RS232 / RS485 Full Duplex / Half Duplex
+   - CAN: 1x CAN FD
+   - Digital I/O: via Expansion Connector
+   - MMX/SD/SDIO: microSD slot
+   - Display: LVDS(1x4 or 1x8), MIPI DSI(1x4), HDMI
+   - Audio: SAI
+   - Camera: 1x MIPI CSI-2 (PhyCAM-M), 1x Parallel
+   - Expansion Bus: I2C, SPI, SDIO, UART, USB
+
+- Debug:
+
+   - JTAG 10-pin connector
+   - USB-C for UART debug, 2x serial ports for A55 and M33
+
+
+.. image:: img/phyboard_nash.webp
+   :width: 720px
+   :align: center
+   :height: 405px
+   :alt: phyBOARD Nash
+
+More information about the board can be found at the `PHYTEC website`_.
+
+Supported Features
+==================
+
+The phyBOARD Nash Cortex-A Core configuration supports the following hardware
```

Comment:
should have the exact name used to build using west/cmake and label as board target not board configuration

Reply:
Right, that gives a better overview, I changed that.